source/content/posts/custom-share-intents.md

```diff
@@ -0,0 +1,178 @@
+
+![image](../custom-share-intents/img_header.png)
+
+<p style="text-align: justify;">
+El articulo se centra en poder personalizar la lista de ShareIntents en tu dispositivo móvil para poder compartir algún tipo de Datos (Imágenes, Texto Plano, enlaces, etc.) a través de dichas aplicaciones o redes sociales. 
+</p>
+
+## ¿Cuál es la novedad?
+<p style="text-align: justify;">
+Hasta este punto se podría hacer uso de los componentes nativos que el sistema operativo nos proporciona, pero que pasa si deseamos compartir cierta información solo con algunas aplicaciones especificas..?, ¿ Agregar algún componente o efecto a la hora de mostrar la lista..?, ¿Personalizar los ítems de la lista a mi antojo..?, ¿Cambiar el tipo de fuente de la lista..?.
```

Comment:
I have some suggestions for you:
*especificas..? , two points or three points?
* ¿ Agregar -> ¿Agregar
*